src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws InterruptedException {
```

Comment:
Remove "sleep"
Remove empty lines
Remove "throws
```suggestion
    public static void main(String[] args) {
```

Reply:
done